specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -904,6 +904,10 @@
           "$ref": "#/definitions/WorkspaceCustomParameterType",
           "description": "The type of variable that this is"
         },
+        "entities": {
+          "$ref": "#/definitions/ManagedServicesEncryptionEntities",
+          "description": "The value which should be used for this field"
```

Comment:
You need to add identity information as well.

Reply:
Are you sure? There is no identity field in the Milestone 1 spec in the design document.

Reply:
Yes get the review approved for M5 as well, we can merge the PR once both M1 and M5 changes are deployed. Having to get approval second time is tough

Reply:
Ashwin today we discussed and decided we will try to ship the Public Preview scope related stuff in current version and all the MI related stuff in new API version. As per the discussion, we shouldn't have identity in this version, please remove it.